### ISSUE TITLE
How to use the p(y|X) to obtain the y*?

### ISSUE THREAD

**Issue by jichaofeng** [tjiiv-cprg/EPro-PnP#19]
Thank you for sharing your code. During inference, when we obtain the pose distribution p(y|X) by the probabilistic PnP layer, how to use the p(y|X) to obtain the y*?  We can not understand the Section 3.4.

**Comment by jichaofeng** [comment]
Thank you for your answer. y* = argmax p(y|X), which is equivalent to solving the optimization problem in Eq. (1). Are these two optimization function is equivalent? Please excuse my poor math.

**Comment by Lakonik** [comment]
Yes they are equivalent. y* = argmax p(y|X) = argmin -log p(y|X) = argmin -log p(X|y) + ~~const~~

**Comment by Lakonik** [comment]
Yes that's right
